free-2.md

```diff
@@ -10,42 +10,104 @@ good way to meet their needs.
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
-
+My domain specific language would help people who play fantasy sports games,
+most specifically (fantasy soccer)[https://fantasy.premierleague.com/].
+Oftentimes, fantasy sports leagues are created with entrance fees and prize 
+money involved. Therefore, it is important to keep track of your squad and team
+throughout the season. However, it is nearly impossible to _always_ substitute
+injured or unavailable players before each gameweek because of the inherent
+nature of time: injuries can happen at any minute, and we can't keep constantly
+check our fantasy sports. I propose creating a DSL that a) allows people to 
+create the most optimized team selection at the beginning of the season, and b)
+substitute and trade players in their squad automatically.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
-
+A DSL is appropriate because fantasy sports are very click-heavy and it does
+not make sense for it to be. Optimization problems where you are given limited
+funds and asked to make the best decisions are solveable by computing and
+mathematics, so there should be some automation and algorithms involved despite
+the "any given Sunday" nature of sports. Through a language, it is
+possible to ask and answer more optimization questions. It's also possible to
+run a script at particular time intervals to check for injuries and such.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
-
+I was an avid fantasy sports player, but I haven't participated recently
+because I was losing my entrance fee every single year without ever winning,
+largely because I couldn't discipline myself to check the fantasy league
+consistently. Therefore, I think it makes a lot of sense to automate fantasy
+sports for the mundane things like substituting injured players. Furthermore, I
+find it ridiculous that there isn't a way to mine the fantasy database such that
+I can create the best team.
 
 ### Domain
 _Describe the project's domain in five words._
-
+Automating and optimizing fantasy sports. 
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
+I imagine the language would look similar to the following: 
+```
+account = new Account(user, authToken)
+
+account.OptimizedTeamSelection(
+	Budget: 100m, 
+	NumberPlayers: 15,
```

Comment:
How would the players be designed? I assume that players have individual stats to consider when optimizing your team